src/libraries/System.Net.Security/System.Net.Security.sln

```diff
@@ -1,4 +1,4 @@
-Microsoft Visual Studio Solution File, Format Version 12.00
+﻿Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Would changing the BOM marker change anything?

Reply:
Unsure, but I don't care much assuming we won't change these files again anytime soon (now that there is no target framework in them anymore).